lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1416,8 +1418,9 @@ def _resource_sku_to_capability(  # noqa: C901
                     nvme.disk_count = int(node_space.core_count / 8)
                     node_space.features.add(nvme)
             elif name == "MaxDataDiskCount":
+                node_space.disk.max_data_disk_count = int(sku_capability.value)
```

Comment:
Split each meaningful individual change to commit as much as possible. It helps people to understand changes better. We spend a little bit more time during development, like organize changes, add more comments, it's investment to future.

Reply:
I will try to separate them into individual commits. The current code had multiple changes in same file which were related so I had put them in single commit

Reply:
You can use `git add -p <file name>` to commit partial changes of a file. I did that a lot.